pyqir/pyqir/_simple.py

```diff
@@ -84,6 +85,16 @@ def builder(self) -> Builder:
         """The instruction builder."""
         return self._builder
 
+    @property
+    def entry_point(self) -> Function:
+        """The entry point function (automatically generated)."""
+        return self._entry_point
+
+    @property
+    def entry_block(self) -> BasicBlock:
+        """The first basic block of the entry point (automatically generated)."""
+        return self._entry_block
+
```

Comment:
This is used in the test I added, but it should be pretty useful to users too if they are manually messing with basic blocks